CHANGELOG.md

```diff
@@ -4,28 +4,22 @@ All notable changes to this project will be documented in this file.
 
 ## [x.x.x] - UNRELEASED
 
-### Overview
-
-### EDC UI
-
-### EDC-Extensions
-
-#### Major Changes
-
-#### Minor Changes
-
-#### Patch Changes
-
-### Deployment Migration Notes
+### Major Changes
 
-#### Compatible Versions
+- Upgrade to core-EDC version `0.2.0`
+- Now using the `Dataspace Protocol`
+- Major changes to the management API
+  - Examples for the new requests are located in the postman collection in the `docs` folder
+  - The `OpenAPI` file has been updated to represent the EDC version `0.1.2`
 
-- Connector Backend Docker Images:
-  - Dev EDC: `ghcr.io/sovity/edc-dev:{{ VERSION HERE }}`
-  - Sovity EDC CE: `ghcr.io/sovity/edc-ce:{{ VERSION HERE }}`
-  - MDS EDC CE: `ghcr.io/sovity/edc-ce-mds:{{ VERSION HERE }}`
-- Connector UI Docker Image: `ghcr.io/sovity/edc-ui:0.0.1-milestone-8-sovity{{ VERSION HERE }}`
+### Migration Notes
 
+1. The `MY_EDC_IDS_BASE_URL` has been renamed to `MY_EDC_PROTOCOL_BASE_URL`
+2. The default value of `WEB_HTTP_PROTOCOL_PATH` been changed from `${MY_EDC_BASE_PATH}/api/v1/ids` to `${MY_EDC_BASE_PATH}/api/v1/dsp`
+3. New environment variable: `EDC_PARTICIPANT_ID`: `provider`
+4. New environment variable: `EDC_JSONLD_HTTPS_ENABLED`: `true`
+5. New environment variable: `EDC_DSP_CALLBACK_ADDRESS`: `http://edc:11003/api/v1/dsp`
+6. `v1` Management API has been deprecated in favor of the `JSON-LD` `v2` Management API. All endpoints have a `v2` prefix now (example: `http://localhost:11002/api/v1/management/assets/request` is now available at `http://localhost:11002/api/management/v2/assets/request`)
```

Comment:
Just to comment somewhere: my impression is that you can't say that across all API groups of the management-api anymore with v0.2.0 "All endpoints have a `v2` prefix now". In v0.2.0 there is already a /v3 for assets, for example the /assets/request mentioned is on v3 with v0.2.0 (= /v3/assets/request), whereas the mentioned /v2 has been marked as deprecated. As a result, we now have a mix of v2 and v3 APIs per management-API-group.

![image](https://github.com/sovity/edc-extensions/assets/75306992/61338a97-2419-4699-9f81-a62020efcbe4)

https://app.swaggerhub.com/apis/eclipse-edc-bot/management-api/0.2.0#/Asset/updateAsset